### PR TITLE
Bump references version

### PR DESCRIPTION
**Pull request by chitang**

### CODE REVIEW

File: common-pg/src/main/scala/com/socrata/pg/store/PGSecondaryUniverse.scala [hunk 1]
```diff
@@ -91,7 +91,7 @@ class PGSecondaryUniverse[SoQLType, SoQLValue](
 
   lazy val datasetMapWriter: PGSecondaryDatasetMapWriter[SoQLType]  =
     new PGSecondaryDatasetMapWriter(conn, typeContext.typeNamespace,
-      timingReport, obfuscationKeyGenerator, initialCounterValue)
+      timingReport, obfuscationKeyGenerator, initialCounterValue, 0L)
```
Comment (rjmac) [inline common-pg/src/main/scala/com/socrata/pg/store/PGSecondaryUniverse.scala]: This 0 should probably come from the same place initialCounterValue comes form

File: common-pg/src/main/scala/com/socrata/pg/store/SoQLCommon.scala [hunk 2]
```diff
@@ -58,6 +58,8 @@ class PostgresUniverseCommon(val tablespace: String => Option[String],
 
   val repFor = repForIndex
 
+  val initialLatestDataVersion: Long = 0
```
Comment (rjmac) [inline common-pg/src/main/scala/com/socrata/pg/store/SoQLCommon.scala]: ..which presumably is here?  This isn't used otherwise.

**Comment by chitang** [comment]
migration can be problematic?

**Comment by rjmac** [comment]
Yes, it'll take a lock on an extremely-frequently-used table, so this'll probably have to do the rolling-secondary thing.

**Comment by rjmac** [comment]
Actually it won't take a _read_ lock, so maybe it'll be fine?

**Pull request merged**